pkg/cloud/services/ec2/instances.go

```diff
@@ -310,16 +310,16 @@ func (s *Service) runInstance(role string, i *infrav1.Instance) (*infrav1.Instan
 	}
 
 	if len(i.NetworkInterfaces) > 0 {
-		instances := make([]*ec2.InstanceNetworkInterfaceSpecification, 0, len(i.NetworkInterfaces))
+		netInterfaces := make([]*ec2.InstanceNetworkInterfaceSpecification, 0, len(i.NetworkInterfaces))
```

Comment:
these were mistakenly called instances I think. updated the name to be more clear that they're network interfaces.